capstone/src/main/webapp/youtubeScript.js

```diff
@@ -51,25 +51,21 @@ async function callYoutube() {
   /** Checks if input follows channel ID format, if not attempts to convert it to channel ID*/
   let response;
   let responseJson;
-  if (channelId[0] == "U" && channelId[1] == "C" && channelId.length == 24 && isLetter(channelId[channelId.length-1])) {
-    response = await fetch('/youtube_servlet?channelId=' + channelId);
-    responseJson = await response.json();
+  if (channelId[0] == "U" && channelId[1] == "C" && channelId.length == 24 && isLetter(channelId[channelId.length-1])) {;
+    responseJson = await callYoutubeServlet(channelId);
     if (responseJson.hasOwnProperty('error')) {
-      inputCommentsToPerspective([]);
       return;
     }
     document.getElementById('search-type').innerHTML = "Channel ID Search";
   } else {
-    const usernameConverterResponse = await fetch('/youtube_username_servlet?channelId=' + channelId);
-    const usernameConverterResponseJson = await usernameConverterResponse.json();
+    const usernameConverterResponseJson = await callYoutubeUsernameServlet(channelId);
+    console.log(usernameConverterResponseJson);
```

Comment:
This console.log line can be deleted

Reply:
addressed in "address nits and add youtube caller support"